i18n/miraheze/qqq.json

```diff
@@ -70,6 +70,7 @@
 	"group-global-ipblock-exempt": "Global IP block exemptions",
 	"group-global-ipblock-exempt-member": "{{GENDER:$1|global IP block exempt}}",
 	"group-member": "Members\n\n{{Identical|Member}}",
+	"group-member-member": "member",
```

Comment:
member\n\n{{Identical|Member}}

Reply:
Why do we have to do this now?

Reply:
This is for documentation usages on translatewiki

Reply:
Oh okay. :)

Reply:
Should be done now. :)